Content/Items/Currency/EchoingShard.cs

```diff
@@ -14,7 +17,7 @@ public override void SetDefaults()
 	
 	public override bool CanRightClick()
 	{
-		return base.CanRightClick() && Main.LocalPlayer.HeldItem.GetInstanceData().Rarity != ItemRarity.Normal;
+		return base.CanRightClick() && Main.LocalPlayer.HeldItem.GetInstanceData().Rarity is not ItemRarity.Unique;
```

Comment:
Hardly a difference, but I have to ask - why `is not` over `!=`?

Reply:
I think I had `is not and` at one point but the `and` went away and I just left it in there

Reply:
Makes sense. There's probably a microoptimization difference but oh well lol